osaca/data/isa/aarch64.yml

```diff
@@ -5,7 +5,17 @@ isa: "AArch64"
 #   mnemonic op1 ... opN
 # means that op1 is the only destination operand and op2 to op(N) are source operands.
 instruction_forms:
-    - name: fmla
+    - name: "cmp"
+      operands:
+        - class: "register"
+          prefix: "x"
+          source: true
+          destination: false
+        - class: "register"
+          prefix: "x"
+          source: true
+          destination: false
+    - name: "fmla"
```

Comment:
already in master (see below, starting at line 184). this would be a duplicate entry

Reply:
Resolved in d932572